docs/translations/translation-basics.md

```diff
@@ -0,0 +1,59 @@
+# Translation basics
+
+## Localization functions
+
+Since [WordPress 2.1 "Ella"](https://wordpress.org/support/wordpress-version/version-2-1/), WordPress offers i18n for PHP files, and since [WordPress 5.0 "Bebo"](https://wordpress.org/support/wordpress-version/version-5-0/), it also offers i18n for JS files. Handling translations is pretty straight forward. Both PHP and JS handle translations similar. WordPress offers the functions:
```

Comment:
I know i18n is a well-known abbreviation, but maybe we could write `internationalization (i18n)` just in case someone is unfamiliar with the abbreviation?

Reply:
Great suggestion. I've changed that part accordingly.